practice.html

```diff
@@ -8,12 +8,17 @@
     <title>Document</title>
   </head>
   <body>
-    <input type="text" id="input" />
-    <button onclick="addTask()">ADD</button>
-    <div id="result">
-      <ul id="list"></ul>
+    <div class="container">
+      <h1>To Do List</h1>
+      <div>
+        <input type="text" id="input" oninput="addChangeHandler(value)" />
+        <button onclick="addTask()" id="add" disabled>ADD</button>
+      </div>
+      <div id="result" class="display">
```

Comment:
Rename class name, it should be such that one can clearly understand is the purpose of class.

Reply:
Changed the class names.